autoload/denops/_internal/server/chan.vim

```diff
@@ -60,7 +60,8 @@ endfunction
 
 function! denops#_internal#server#chan#close() abort
   if s:chan is# v:null
-    throw '[denops] Channel does not exist yet'
+    call denops#_internal#echo#debug("Channel does not exist yet")
+    return
   endif
```

Comment:
`denops#_internal#server#chan#close()` is an internal method, so throwing it should let us catch the bug.
No changes should be made here.

Reply:
Agree.

Reply:
Add reverse patch or delete commit 3170593.